### PR TITLE
{lib}[foss/2020a] pandas v1.1.2 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/a4016a1e4ee1b9003cc03d8af0ffd6d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/f910d3c0c0e2c45e03511d2cdac095b3 for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/238122d0de5b51d676099d82d6a7a7db for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node115.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a247dd446ab2751cad3ba2f8be4ee412 for a full test report.

File: easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', versionsuffix),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('numexpr', '2.7.1', {
```
Comment (lexming) [inline easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb:23]: We have specific easyconfigs for `numexpr` and version 2.7.1 is the one used in `2020a`. So I would rather add `numexpr-2.7.1-foss-2020a-Python-3.8.2.eb` and depend on it, than bundle `numexpr` here.

File: easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        'checksums': ['b0d239d9827e1bcee08344fd05835823bc60aff97232e35a928214d03ff802b1'],
+    }),
+    ('Bottleneck', '1.3.2', {
```
Comment (lexming) [inline easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb:26]: Same for Bottleneck, we have specific easyconfigs for it. Would you mind adding `Bottleneck-1.3.2-foss-2020a-Python-3.8.2.eb` and depend on it?

**Review by lexming — CHANGES_REQUESTED** [review]
I favour adding easyconfigs for `numexpr` and `Bottleneck` instead of bundling. Please check my comments below. Once we have easyconfigs in place for these two packages, we could also update `Scipy-Bundle` to also depend on `numexpr` and `Bottleneck`.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3406.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/e7efbd73d6fbba27eb2239c2548186f9 for a full test report.

File: easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb]: done in 1fc0562

File: easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/p/pandas/pandas-1.1.2-foss-2020a-Python-3.8.2.eb]: done in 1fc0562

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2414.golett.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz (haswell), Python 2.7.5
See https://gist.github.com/d82e847b52b240fd3de9c44f7b9ca266 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/74ce91f1457b66d116fe3b845c380233 for a full test report.

**Review by lexming — APPROVED** [review]
@boegel thanks for the update!

LGTM

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node376.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/8aa4d8d8b917b85e964bd043e0716f3e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node119.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/79f8721296d9dcf556b0918730bcd109 for a full test report.

**Comment by lexming** [comment]
Going in, thanks @boegel !

**Pull request merged**